### PR TITLE
HV: enlarge the CMA size for uos trusty

### PR DESCRIPTION
**Pull request by liuwei987**
sometimes, there would be CMA allocate failue when doing cycle rebooting android uos.
as there are different CMA clients, and android trusty need 16M contiguous memory,
32M CMA size may make this failure easy to come out.
This patch is to enlarge the CMA sizeto 64M, which mitigate the failure.

Signed-off-by: Wei Liu <weix.w.liu@intel.com>
Signed-off-by: Jason Chen CJ <jason.cj.chen@intel.com>

### CODE REVIEW

**Comment by gvancuts** [comment]
Do we need the same change applied to `sos_bootargs_release.txt`?

**Comment by liuwei987** [comment]
Yes, i will update it. 
Thank you @gvancuts

**Comment by gvancuts** [comment]
Thanks!

File: devicemodel/samples/apl-mrb/sos_bootargs_release.txt [hunk 1]
```diff
@@ -1 +1 @@
-console=tty0 i915.nuclear_pageflip=1 root=/dev/mmcblk1p1 rw rootwait quiet loglevel=3 no_timer_check consoleblank=0 i915.enable_initial_modeset=1 i915.tsd_init=7 i915.tsd_delay=2000 video=DP-1:d video=DP-2:d i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_guc_loading=0 i915.enable_guc_submission=0 i915.enable_preemption=1 i915.context_priority_mode=2 i915.enable_gvt=1 i915.enable_guc=0 cma=32M@0-
+console=tty0 i915.nuclear_pageflip=1 root=/dev/mmcblk1p1 rw rootwait quiet loglevel=3 no_timer_check consoleblank=0 i915.enable_initial_modeset=1 i915.tsd_init=7 i915.tsd_delay=2000 video=DP-1:d video=DP-2:d i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_guc_loading=0 i915.enable_guc_submission=0 i915.enable_preemption=1 i915.context_priority_mode=2 i915.enable_gvt=1 i915.enable_guc=0 cma=64M@0-
```
Comment (dongyaozu) [inline devicemodel/samples/apl-mrb/sos_bootargs_release.txt]: Do we update the doc too?

Comment (gvancuts) [reply devicemodel/samples/apl-mrb/sos_bootargs_release.txt]: Unless I've missed it, that parameter (and its value) is not referenced in any of our doc so we should be good here.

**Comment by lijinxia** [comment]
Verified+1

**Pull request merged**